src/components/Calculator.js

```diff
@@ -0,0 +1,62 @@
+// /src/components/Calculator.js
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from './Button';
+
+function Output(props) {
+  const { result } = props;
+  return (
+    <div className="outputContainer">
+      <span className="output">
+        {result}
+      </span>
+    </div>
+  );
+}
+
+Output.defaultProps = {
+  result: '-1',
+};
+
+Output.propTypes = {
+  result: PropTypes.number,
+};
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.calculation = 0;
+  }
```

Comment:
- [ ] Great job working so far but I think you missed to make your calculator component as functional component as I can see here it is class based so please make it functional based as required 😊✨🔥🤩
![image](https://user-images.githubusercontent.com/58553711/219908181-4c747c53-7160-4336-b2f2-47d808b7f84c.png)